docs/developers/guides/automated_coin_conversion.md

```diff
@@ -0,0 +1,112 @@
+	Amount Int    `protobuf:"bytes,2,opt,name=amount,proto3,customtype=Int" json:"amount"`
+}
+```
+
+## Inbound transactions
+
+Your users may want to move IBC Coins from other Cosmos chains onto Evmos.
+To use these IBC coins on dApps deployed on Evmos, they need an ERC-20 representation of these.
+The automated coin conversion feature automatically converts the incoming IBC coins into their ERC-20 representation.
+In this way, you don't need to manually convert the incoming IBC coins into ERC-20 tokens.
+As a result, your users can use the IBC coins as ERC-20 tokens as soon as they arrive to their wallets.
+
+It should be considered that only registered token pairs are converted.
+If the token pair is not registered, users will receive the corresponding IBC coin on their wallet without any further changes.
+
+:::tip
+**Note**: If your users have IBC coins on Evmos already, and they receive an IBC transfer in the denomination of an already registered token pair,
+their **whole balance** will be converted (the current balance plus the transfer amount).
+:::
+
+## FAQ
+
+### How do I send an ERC-20 via IBC?
+
+With the new automated coin conversion feature, you can send ERC-20 via IBC right away.
+The conversion step is done automatically under the hood.
+To do this operation you only need to specify the corresponding denomination on the `MsgTransfer` struct.
+For example, if we want to send an ERC-20 token called `TestCoin` via IBC, use `Token.Denom = "TestCoin"`.
+
+### Can I send WEVMOS to other chains?
+
+WEVMOS transfers are not supported at the moment.
+However, you can unwrap manually the WEVMOS tokens using the [Evmos dashboard](https://app.evmos.org/assets) or [Diffusion](https://app.diffusion.fi/).
+Then you can perform a regular IBC transfer using the Evmos tokens.
+
+### Does automated coin conversion apply to all coins?
+
+The automated coin conversion covers all IBC coins and ERC-20 tokens as long as the appropriate token mapping was registered through governance.
+If the token pair is not registered, the IBC coin will be left as is.
+Additionally, Evmos token conversion is not automated.
+Considering that the Evmos token is used for staking and paying gas fees, the team decided to exclude the native token automated conversion.
+Thus, the user experience is not undermined by this feature.
+
+### How do I convert the Evmos token to ERC-20?
+
+The conversion from EVMOS token to WEVMOS is not automated.
+If you want to convert EVMOS tokens into its ERC-20 representation, you will need to use [the Assets page](https://app.evmos.org/assets).
+
+### Do I still need to use [the Assets page](https://app.evmos.org/assets)?
+
+Yes! If you want to convert Evmos tokens into their ERC-20 representation, you will need to do it manually on [the Assets page](https://app.evmos.org/assets).
+Evmos token automated conversion was excluded in this feature to avoid damaging user experience.
```

Comment:
> Evmos token automated conversion was excluded in this feature to avoid damaging user experience.

the reason why it was excluded was that the EVMOS token also has utility for staking, governance and paying for gas on the EVM